installation.md

```diff
@@ -152,11 +152,55 @@ with `M-x company-mode`.
 - **Debug logs**: available in the `EGLOT events` buffer.
 - **Command-line flags and alternate binary**: instead of adding `"clangd"`
   to `eglot-server-programs`, add `("/path/to/clangd" "-log=verbose")` etc.
+
+---
+
+[lsp-mode](https://github.com/emacs-lsp/lsp-mode) can also be configured to work with clangd. It has support for all standard clangd features and some of the extensions, like completion scores, or type hierarchies.
+
+Install lsp-mode with `M-x package-install RET lsp-mode RET`.
+
+Add the following to `~/.emacs` to enable lsp-mode:
+
+```
+(require 'lsp-mode)
+(add-hook 'c-mode-hook #'lsp)
+(add-hook 'c++-mode-hook #'lsp)
+```
+
+### Indexing status
```

Comment:
This is nice but it's not really a feature needed to get started and the user should discover it without any action needed (if i'm understanding correctly). So I'm not sure we need to mention this.

Reply:
I've removed it.